src/calculateRentalCost.js

```diff
@@ -4,7 +4,15 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const pricePerDay = 40;
+
+  if (days < 3) {
+    return pricePerDay * days;
+  } else if (days >= 3 && days <= 6) {
+    return pricePerDay * days - 20;
+  }
+
+  return pricePerDay * days - 50;
```

Comment:
The function name 'calculateRentalCost' is descriptive and clear, which is good. However, the function uses hard-coded literals for the price per day, and discount values, which should be replaced with named constants to improve code readability and maintainability.